unison-src/transcripts/squash.output.md

```diff
@@ -95,18 +95,16 @@ Let's try again, but using a `merge.squash` (or just `squash`) instead. The hist
 
   Nothing changed as a result of the merge.
 
-  😶
-  
-  builtin2 was already up-to-date with mybuiltin.
```

Comment:
Interesting that this was removed; not sure what's up here but I'll look into it.